### DISCUSSION TITLE
Question: Tracking outbound links... on the same domain?

### DISCUSSION THREAD

**Discussion by MarcusOtter** [plausible/analytics#1825] [Anything Else]
Hi! First of all thank you so much for this wonderful service. It's been extremely easy to set up with a few websites and the docs are always on-point. Really great experience so far!

I have a specific use case that I can't figure out how to solve. Basically, my goal is to track outbound links. You have some [really nice docs](https://plausible.io/docs/outbound-link-click-tracking) for this and it works great as long as the link is actually, well, outbound, and not on the same domain as seen [here](https://github.com/plausible/analytics/blob/master/tracker/src/plausible.js#L96). However, I am using [Netlify redirects](https://docs.netlify.com/routing/redirects/) for my links, so I can do `https://example.com/github` which would redirect me to my GitHub page with a `301 Permanently Moved`. However, when I click something like `<a href="/github">GitHub</a>`, plausible does not recognize this as an outbound link, and therefore doesn't count it. It does work if I replace it with the entire `github.com/...` URL, but I would prefer to use the redirects instead.

My question boils down to this: **What is the easiest way for me to track outbound link clicks when they are redirects from the same domain?**

I have looked around at other features such as custom events and [link clicks tracking](https://plausible.io/docs/custom-event-goals#code-example-for-tracking-link-clicks) with the `data-analytics` attribute, ~~but it doesn't seem to work for me~~. As I was writing this, it started working all of a sudden. Don't know what I changed, nothing I think. So, my current solution is to add [the snippet](https://plausible.io/docs/custom-event-goals#2-add-the-javascript-that-will-be-sending-the-link-click-events-to-plausible) listed in the docs and change the link to `<a href="/github" data-analytics='"GitHub"'>GitHub</a>` as well as specifying a Custom event matching the names of each link in the site settings. I guess it works, but it's a bit tedious to manually create an event for every possible link, as well as adding the additional snippet and specifying an HTML attribute on each link. All of the events are also spread out and not grouped under a heading like "Outbound link click" would be, but I guess there's probably a way for me to make a custom event like that and then modify the snippet to post to that custom event and then include the link in the custom props. But at least what I have now works, so the question sort of resolved itself. Though it would still be interesting to know if there's an easier way! :)

**Comment by ukutaht** [comment]
@MarcusOtter Thanks for the write-up. Supplying an easy way to tag links (and forms) is coming up in our roadmap very soon. Stay tuned :)
